azurelinuxagent/ga/cgroupconfigurator.py

```diff
@@ -140,9 +140,8 @@ def initialize(self):
                     # If a distro is not supported, attempt to clean up any existing drop in files in case it was
                     # previously supported. It is necessary to cleanup in this scenario in case the OS hits any bugs on
                     # the kernel related to cgroups.
-                    if not self.using_cgroup_v2():
-                        log_cgroup_info("Agent will reset the quotas in case cgroup usage went from enabled to disabled")
-                        self._reset_agent_cgroup_setup()
+                    log_cgroup_info("Agent will reset the quotas in case cgroup usage went from enabled to disabled")
+                    self._reset_agent_cgroup_setup()
```

Comment:
in _reset_agent_cgroup_setup, could we add an info message when we reset the cpu quota?

also, should we always call _reset_cpu_quota, even if there are no dropin files?

```
                    if len(files_to_cleanup) > 0:
                        log_cgroup_info("Found drop-in files; attempting agent cgroup setup cleanup", send_event=False)
                        self._cleanup_all_files(files_to_cleanup)
                        self._reset_cpu_quota(systemd.get_agent_unit_name())

```

Reply:
`in _reset_agent_cgroup_setup, could we add an info message when we reset the cpu quota?
`
what info message, the reset_cpu_quota already have info messgaes

`also, should we always call _reset_cpu_quota, even if there are no dropin files?
`
This will be called on every distro and we don't know if reset works and any side effects of touching cgroups. I understand where you are coming from, if we used systemctl for setting quota and at some point, due to issues and we drop the cgroup support, we should reset that quota with systemctl